administrator/components/com_templates/Helper/RenderHelper.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_templates
+ *
+ * @copyright   Copyright (C) 2005 - 2019 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Templates\Administrator\Helper;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Plugin\PluginHelper;
+
+/**
+ * Template Helper class.
+ *
+ * @since  4.0
+ */
+abstract class RenderHelper
+{
+	/**
+	 * Decode pagebuilder param and render the elements
+	 *
+	 * @param   string $param JSON with grids, columns and modules, build by the pagebuilder
+	 *
+	 * @return  string
+	 *
+	 * @since   4.0
```

Comment:
`@since  __DEPLOY_VERSION__` Because we don't know when this feature merge in Joomla.